File: frozen_eq/plot_replicate_hoffman.m
clear
 figure(999)
 
 load He_0_3_frozen_eq
 dlmwrite('He_0_3_frozen_eq_Tbeam.dat',Tbeam_nadir,'\t')
 plot(Tbeam_nadir,'+b')
 hold on;
 
 load He_0_34_frozen_eq
 dlmwrite('He_0_34_frozen_eq_Tbeam.dat',Tbeam_nadir,'\t')
 plot(Tbeam_nadir,'+k')
 
 load He_0_9_frozen_eq
 dlmwrite('He_0_9_frozen_eq_Tbeam.dat',Tbeam_nadir,'\t')
 plot(Tbeam_nadir,'+g')
 
 load He_0_11_frozen_eq
 dlmwrite('He_0_11_frozen_eq_Tbeam.dat',Tbeam_nadir,'\t')
 plot (Tbeam_nadir,'+r')
 
 load He_0_3_frozen_eq_auto
 dlmwrite('He_0_3_frozen_eq_auto_Tbeam.dat',Tbeam_nadir,'\t')
 plot (Tbeam_nadir, 'ob')

 load He_0_34_frozen_eq_auto
 dlmwrite('He_0_34_frozen_eq_auto_Tbeam.dat',Tbeam_nadir,'\t')
 plot (Tbeam_nadir,'ok')
 
  
 load He_0_9_frozen_eq_auto
 dlmwrite('He_0_9_frozen_eq_auto_Tbeam.dat',Tbeam_nadir,'\t')
 plot (Tbeam_nadir, 'og')
 
 load He_0_11_frozen_eq_auto
 dlmwrite('He_0_11_frozen_eq_auto_Tbeam.dat',Tbeam_nadir,'\t')
 plot (Tbeam_nadir, 'or')
 
 set(gca,'XTick',1:1:15)
 set(gca,'XTickLabel',Model_names)
 legend('2\,km Step 3% He','2\,km Step 3.4% He','2\,km Step 9% He','2\,km Step 11% He', ...
        'AutoStep 3% He','AutoStep 3.4% He','AutoStep 9% He','AutoStep 11% He')
 title('Nadir Brightness Temperatures')           
 hold off
 
 figure(1000)
 load He_0_3_frozen_eq
 dlmwrite('He_0_3_frozen_eq_residual.dat',residual,'\t')
 plot(residual,'+b')
 
 hold on;
 
 load He_0_34_frozen_eq
 dlmwrite('He_0_34_frozen_eq_residual.dat',residual,'\t')
 plot(residual,'+k')
 
 load He_0_9_frozen_eq
 dlmwrite('He_0_9_frozen_eq_residual.dat',residual,'\t')
 plot(residual,'+g')
 
 load He_0_11_frozen_eq
 dlmwrite('He_0_11_frozen_eq_residual.dat',residual,'\t')
 plot (residual,'+r')

 load He_0_3_frozen_eq_auto
 dlmwrite('He_0_3_frozen_eq_auto_residual.dat',residual,'\t')
 plot (residual, 'ob')
 
 load He_0_34_frozen_eq_auto
 dlmwrite('He_0_34_frozen_eq_auto_residual.dat',residual,'\t')
 plot (residual,'ok')
 
 load He_0_9_frozen_eq_auto
 dlmwrite('He_0_9_frozen_eq_auto_residual.dat',residual,'\t')
 plot (residual, 'og')
 
 load He_0_11_frozen_eq_auto
 dlmwrite('He_0_11_frozen_eq_auto_residual.dat',residual,'\t')
 plot (residual, 'or')
 
 set(gca,'XTick',1:1:15)
 set(gca,'XTickLabel',Model_names)
 legend('2\,km Step 3% He','2\,km Step 3.4% He','2\,km Step 9% He','2\,km Step 11% He', ...
        'AutoStep 3% He','AutoStep 3.4% He','AutoStep 9% He','AutoStep 11% He')
 title('Nadir Brightness Differences Comparing against Hoffman')           
 hold off
 
 
 
 
 
% 
% figure(1000)
% plot(Tbeam_seventy_five-Tbeam_thesis_seventy_five,'+k')
% hold on;
% plot(Tbeam_nadir-Tbeam_thesis,'+b')
% set(gca,'XTickLabel',Model_names)
% set(gca,'XTick',1:1:15)
% set(gca,'XTickLabel',Model_names)
% legend('Error in Limb values ^{\circ} K','Error in nadir values ^{\circ} K')
% title('Error compared with Hoffman')
% %savefig('error','pdf','-cmyk','-r600')
% hold off
% 
% figure(1001)
% plot(Tbeam_thesis-Tbeam_thesis_seventy_five,'+r')
% hold on
% plot(Tbeam_nadir-Tbeam_seventy_five,'+g')
% set(gca,'XTick',1:1:15)
% set(gca,'XTickLabel',Model_names)
% legend('Hoffman Values','Run values')
% title('Compare Limb darkening')
% %savefig('darkening','pdf','-cmyk','-r600')
% hold off
% %mv *.pdf plots
% /*P T dr XH2 XHe XH2S XNH3 XH2O XCH4 XPH3  clouds DNH4SH DH2S DNH3 DH2O DCH4
% DPH3 DSOL g mu refr_w/o refr_ w/  */
color_funtime={'b','g','r','k','y','--b','--g','--r','--k','--y','+b','+g','+r','+k','+y'}
figure(1002)
for j=1:1
        plot(XPH3(1:296,1),tcme(1:296,1,1),color_funtime{j})
        hold on;
end
hold off

File: frozen_eq/plot_replicate_hoffman.py
 #!/sw/bin/python2.3
from pylab import * 
matplotlib.rc('text',usetex= True)
matplotlib.use('PS')
Tbeam_nadir=load('He_0_3_frozen_eq_Tbeam.dat')
plot(Tbeam_nadir,'+b')
#hold on;
 
Tbeam_nadir=load('He_0_34_frozen_eq_Tbeam.dat')
plot(Tbeam_nadir,'+k')
 
Tbeam_nadir=load('He_0_9_frozen_eq_Tbeam.dat')
plot(Tbeam_nadir,'+g')
 
 
Tbeam_nadir=load('He_0_11_frozen_eq_Tbeam.dat')
plot (Tbeam_nadir,'+r')
 
 
Tbeam_nadir=load('He_0_3_frozen_eq_auto_Tbeam.dat')
plot (Tbeam_nadir, 'xb')

Tbeam_nadir=load('He_0_34_frozen_eq_auto_Tbeam.dat')
plot (Tbeam_nadir,'xk')
 
Tbeam_nadir=load('He_0_9_frozen_eq_auto_Tbeam.dat')
plot (Tbeam_nadir, 'xg')
 
Tbeam_nadir=load('He_0_11_frozen_eq_auto_Tbeam.dat')
plot (Tbeam_nadir, 'xr')
 
#set(gca,'XTick',:)
xticks(arange(15),('A','A2','A3','A4','B','B2','B3','C','C2','C3','C4','D','E','F','G'))
legend(('2\,km Step 3\% He','2\,km Step 3.4\% He','2\,km Step 9\% He','2\,km Step 11\% He','AutoStep 3\% He','AutoStep 3.4\% He','AutoStep 9\% He','AutoStep 11\% He'),loc='best',numpoints=1)
xlabel('Model Names')
ylim(140,150)
ylabel('T ( $^{\circ}$K)')
title('Nadir Brightness Temperatures')           
#hold off
savefig('BT.ps',dpi=600)
 
figure(2)
#load He_0_3_frozen_eq
residual=load('He_0_3_frozen_eq_residual.dat')
plot(residual,'+b')
 
#hold on;

#load He_0_34_frozen_eq
residual=load('He_0_34_frozen_eq_residual.dat')
plot(residual,'+k')
 
#load He_0_9_frozen_eq
residual=load('He_0_9_frozen_eq_residual.dat')
plot(residual,'+g')
 
#load He_0_11_frozen_eq
residual=load('He_0_11_frozen_eq_residual.dat')
plot (residual,'+r')

#load He_0_3_frozen_eq_auto
residual=load('He_0_3_frozen_eq_auto_residual.dat')
plot (residual, 'xb')
 
#load He_0_34_frozen_eq_auto
residual=load('He_0_34_frozen_eq_auto_residual.dat')
plot (residual,'xk')
 
#load He_0_9_frozen_eq_auto
residual=load('He_0_9_frozen_eq_auto_residual.dat')
plot (residual, 'xg')
 
#load He_0_11_frozen_eq_auto
residual=load('He_0_11_frozen_eq_auto_residual.dat')
plot (residual, 'xr')
 
#set(gca,'XTick',1:1:15)
xticks(arange(15),('A','A2','A3','A4','B','B2','B3','C','C2','C3','C4','D','E','F','G'))
xlabel('Model Names')
ylabel('$\Delta$ T ($^{\circ}$K)')
ylim(0,8)
#set(gca,'XTickLabel',Model_names)
legend(('2\,km Step 3\% He','2\,km Step 3.4\% He','2\,km Step 9\% He','2\,km Step 11\% He','AutoStep 3\% He','AutoStep 3.4\% He','AutoStep 9\% He','AutoStep 11\% He'),loc='best',numpoints=1)
title('Difference vs. Hoffman 2001')           
savefig('res.ps',dpi=600)
